assets/css/styles.css

```diff
@@ -107,6 +107,259 @@ a {
 .icons:hover {
   transform: translateY(10px);
   transition: 0.5s;
+}
+
+#work-section {
+  display: block;
+  margin-top: 46px;
+  padding: 1em 1em;
+}
+
+.portfolio {
+  width: 95%;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-flow: row;
+  justify-items: center;
+  align-items: center;
+}
+
+.project {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 50px auto;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.work-img img {
+  width: 100%;
+}
+
+.work-description {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.work-title {
+  text-transform: capitalize;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.canopy {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 70%;
+  height: 32px;
+  padding: 0;
+  margin: 12px 0;
+}
+
+.canopy li {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.work-details {
+  height: 76px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.work-tools {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  width: 30%;
+  height: 24px;
+  list-style: none;
+  margin-bottom: 10px;
+}
+
+.work-tools li {
+  padding: 0 12px;
+  color: #6070ff;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.btn {
+  width: 126px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  color: #396df2;
+  margin-bottom: 10px;
+  cursor: pointer;
+}
```

Comment:
- [ ] Kindly consider adding the pressed state button styling here as suggested in the figma design. you can change the color of button when pressed using the `:focus` pseudoclass. kindly consider applying this on all buttons.
figma design guidelines
![as](https://user-images.githubusercontent.com/96683843/234928331-ed5c9c1d-d22a-4820-9886-49f93e391c7d.jpg)